momento-sdk/src/intTest/java/momento/sdk/SimpleCacheClientTest.java

```diff
@@ -9,11 +9,7 @@
 import momento.sdk.exceptions.InvalidArgumentException;
 import momento.sdk.exceptions.NotFoundException;
 import momento.sdk.exceptions.ServerUnavailableException;
-import momento.sdk.messages.CacheDeleteResponse;
-import momento.sdk.messages.CacheGetResponse;
-import momento.sdk.messages.CacheSetResponse;
-import momento.sdk.messages.CreateCacheResponse;
-import momento.sdk.messages.FlushCacheResponse;
+import momento.sdk.messages.*;
```

Comment:
I don't mind in tests much, but we want to avoid wildcard imports if possible. I just increase the class/name count before '*' to 99 in settings > editor > code style > java > imports.

Reply:
I agree. Will correct this in the next commit.